examples/Api.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import metadata from 'component-metadata!react-big-calendar/Calendar';
-import transform from 'lodash/object/transform';
+import { transform } from 'lodash/object';
 
```

Comment:
The destructured imports are a lot clearer because:

1. They are more terse
2. `import { something } from 'package'` would immediately cause your code to break if `something` doesn't exist, whereas `import somethingThatDoesntExist from 'package'` would be more of a silent failure somewhere in your code.

I don't consider adding one dev dependency and one plugin to the `babelrc` "an awful lot of work" for such a clear advantage.